ElementX/Sources/Application/AppCoordinator.swift

```diff
@@ -733,6 +733,51 @@ class AppCoordinator: AppCoordinatorProtocol, AuthenticationFlowCoordinatorDeleg
         }
     }
     
+    private func setupSentry() {
+        SentrySDK.start { options in
+            #if DEBUG
+            options.enabled = false
+            #endif
+            
+            options.dsn = ServiceLocator.shared.settings.bugReportSentryURL.absoluteString
```

Comment:
I think we have a local property for this one.
```suggestion
            options.dsn = appSettings.bugReportSentryURL.absoluteString
```

Reply:
This way I don't need to access self in the block anymore